pdf_parser/pdf_parser.airplane.ts

```diff
@@ -0,0 +1,48 @@
+import airplane from "airplane";
+
+const crawler = require("crawler-request");
+
+export default airplane.task(
+  {
+    slug: "demo_pdf_parser",
+    name: "PDF parser",
+    description:
+      "This task takes an uploaded PDF and parses it into structured data.",
+    parameters: {
+      pdf_upload: {
+        type: "upload",
+        name: "PDF upload",
+        description: "The uploaded PDF file to be parsed",
+      },
+    },
+    // Uncomment to use your own credentials from config variables.
+    // Get your OpenAI API key here: https://platform.openai.com/account/api-keys
+    // How to use config variables in Airplane: https://docs.airplane.dev/platform/configs
+    // envVars: {
+    // OPENAI_API_KEY: { config: "OPENAI_API_KEY" },
+    // },
+  },
+  async (params) => {
+    const response = await crawler(params.pdf_upload.url);
+    const pdfText = response.text;
+
+    const token = process.env.OPENAI_API_KEY ?? "";
+
+    // Return if OPENAI_API_KEY is not set.
+    // Delete this once you create a config variable for OPENAI_API_KEY and
+    // set the environment variable.
```

Comment:
```suggestion
    // Return if OPENAI_API_KEY is not set.
```
Same comment as above